doc/source/report/dynamics.rst

```diff
@@ -3,60 +3,149 @@
+               8.92857143e-02],
+            [  0.00000000e+00,   0.00000000e+00,   5.00000000e-01,
+               5.00000000e-01],
+            [  0.00000000e+00,   1.64835165e-02,   0.00000000e+00,
+               9.83516484e-01]]),
+      ...
+      ...
 
-Data Requirements Description
------------------------------
 
--  We need labels of states of interest with respect to the time intervals in a specific mouse day to extract the structure of the Markov Chain.
 
-Methodology/ Approach Description
----------------------------------
+4. Simluation: 
 
-1. Defining states of interest: which behaviors of the mouse are we interested in? 
+Use the Transition Model generated to simluate a typical mouse day for a typical strain. The simulation is time sensitive which means the simulation mouse is generated depend on the time interval chosen in the transition model. The time interval length is optimized in the future analysis so as the generate the best fake mouse for each strain. 
 
-   - Feeding: labeled by event F
-   - Drinking: labeled by event W
-   - Other active state behaviors: This could possibly include all other movements in the AS state of a mouse besides drinking and eating.
-   - Inactive State: labeled by event I.
+::
 
+    trans_matrix = get_prob_matrix_list(time_df=strain_df,interval_length=1000)
+    mcmc_simulation(trans_matrix, n_per_int=1000)
+    # array([0, 0, 0, ..., 0, 0, 0])
 
-2. Data Preprocessing: convert the data we had into strings of the events chosen. This could be done by checking out the states of a typical mouse day at a lot of equally spaced time points and store the states and the time points in the same order. In order to perform this task, we need basically do the following two steps:
 
-   -  Data cleaning: Clean the raw intervals given by the measurements in the experiment into interval data that makes more sense and consistent. Also need to check if any of our states overlapped.
-   -  Data reformating: Convert the cleaned interval data into strings of events or matrices containing both information from timestamps and the events at those timepoints.
+5. Evaluation System:
 
-3. Estimating Transition Probability: Estimate the transition probability matrix of the Markov Chain using the data given. One of the key challenges to estimate the transition matrix is that our model is actually time continuous non-homogeneous Markov Chain, and the parameters are too difficult to estimate given the data we have. Instead, we figured out a way to make our model a composite of small homogeneous discrete time Markov Chains, so that we could perform a rough estimation of the original time continuous non-homogeneous Markov Chain. We followed the following steps to achieve our goal:
+How to get a score for the performance of our simulation?
+The evaluation system is trying to create a score to evaluate the performance of the simulation. Higher the score, better
```

Comment:
"The higher the score, the better..."

Sorry that all of these are such small corrections, I know it's annoying!